ipfs-cluster/Dockerfile

```diff
@@ -0,0 +1,81 @@
+# referring to ipfs-cluster Dockerfile (https://github.com/ipfs/ipfs-cluster/blob/master/Dockerfile)
+FROM quay.io/cybozu/golang:1.18-focal AS builder
+
+# This dockerfile builds and runs ipfs-cluster-service.
+ENV GOPATH      /go
+ENV SRC_PATH    $GOPATH/src/github.com/ipfs/ipfs-cluster
+ENV GO111MODULE on
+ENV GOPROXY     https://proxy.golang.org
+
+ENV SUEXEC_VERSION v0.2
+ENV TINI_VERSION v0.19.0
+ENV IPFS_CLUSTER_VERSION v1.0.1
+
+RUN apt-get update && apt-get install -y \
+    wget \
+    cmake
+
+RUN set -eux; \
+    dpkgArch="$(dpkg --print-architecture)"; \
+    case "${dpkgArch##*-}" in \
+        "amd64" | "armhf" | "arm64") tiniArch="tini-static-$dpkgArch" ;;\
+        *) echo >&2 "unsupported architecture: ${dpkgArch}"; exit 1 ;; \
+    esac; \
+  cd /tmp \
+  && git clone https://github.com/ncopa/su-exec.git \
+  && cd su-exec \
+  && git checkout -q $SUEXEC_VERSION \
+  && make su-exec-static 
+
+RUN cd /tmp \
+  && git clone https://github.com/krallin/tini.git  \
+  && cd tini \
+  && git checkout refs/tags/$TINI_VERSION \
+  && cmake . \
+  && make \
+  && chmod +x tini
+
+# Get the TLS CA certificates, they're not provided by busybox.
+
```

Comment:
We don't use bybox, so I don't think this comment is necessary.

Reply:
Fixed by 547d342be14e7a08942660da5d9e7149156211eb